### PR TITLE
Finished About page

### PR DESCRIPTION
**Pull request by jonathanpli**
#51

### CODE REVIEW

File: src/main/resources/templates/about.html [hunk 1]
```diff
@@ -0,0 +1,53 @@
+{# About page for exceptionnull #}
+{% extends "base/empty-page" %}
+{% block title %}Exception Null - About{% endblock %}
+
+{% macro person(imgSrc, name, description) %}
+<div class="person">
+    <img src={{ imgSrc }} class="photo"/>
```
Comment (fru1tstand) [inline src/main/resources/templates/about.html]: src="{{ imgSrc }}"

File: src/main/scss/templates/_about.scss [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+      vertical-align: top;
+
+      > img {
+        max-width: 250px;
+        max-height: 250px;
+        border-radius: 50%;
+      }
+
+      .person-desc {
+        font-size: $size-base * 1.5;
```
Comment (fru1tstand) [inline src/main/scss/templates/_about.scss]: font-size in px please.

**Review by fru1tstand — CHANGES_REQUESTED** [review]

File: src/main/resources/templates/about.html (hunk 1 above)
Comment (jonathanpli) [reply src/main/resources/templates/about.html]: done

File: src/main/scss/templates/_about.scss (hunk 2 above)
Comment (jonathanpli) [reply src/main/scss/templates/_about.scss]: done

**Review by fru1tstand — APPROVED** [review]

**Pull request merged**